flink/src/main/java/org/apache/iceberg/flink/TableLoader.java

```diff
@@ -45,23 +46,30 @@ static TableLoader fromCatalog(CatalogLoader catalogLoader, TableIdentifier iden
   }
 
   static TableLoader fromHadoopTable(String location) {
```

Comment:
Is this only visible for testing ?  Moving to test package ?

Reply:
I think it is useful for users, since Flink has default hadoop configuration.